app/controllers/v3/isolation_segments_controller.rb

```diff
@@ -56,6 +60,9 @@ def destroy
     isolation_segment_model = IsolationSegmentModel.where(guid: params[:guid]).first
     resource_not_found!(:isolation_segment) unless isolation_segment_model
 
+    method_not_allowed!('DELETE', "the #{isolation_segment_model.name} Isolation Segment") if
```

Comment:
I think a 405 is exactly the correct response here - from the spec: "The method specified in the Request-Line is not allowed for the resource identified by the Request-URI."  A 405 doesn't mean you can't DELETE any IS, just this one IS.